spec/jobs/clock/refresh_draft_invoices_spec.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Clock::RefreshDraftInvoicesJob, job: true do
```

Comment:
RSpec/FilePath: Spec path should end with `clock/refresh_draft_invoices_job*_spec.rb`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/FilePath)